frontend/pre_analysis.py

```diff
@@ -16,12 +16,20 @@ def __init__(self, prog_ast):
     def maximum_function_args(self):
         """Get the maximum number of function arguments appearing in the AST"""
         func_defs = [node for node in self.all_nodes if isinstance(node, ast.FunctionDef)]
-        return max([len(node.args.args) for node in func_defs] + [1])
+        # A minimum value of 2 for the max number of arguments in the built in functions
+        return max([len(node.args.args) for node in func_defs] + [2])
 
     def maximum_tuple_length(self):
         """Get the maximum length of tuples appearing in the AST"""
         tuples = [node for node in self.all_nodes if isinstance(node, ast.Tuple)]
-        return 0 if not tuples else max([len(node.elts) for node in tuples])
+        # A minimum value of 2 for the max number of arguments in the built in tuples
```

Comment:
would be nice if this wouldn't have to be hard coded; we should be able to get that information from the stubs file, right?